lib/unit/utils/http_helper.rb

```diff
@@ -58,6 +58,10 @@ def self.encode(value, key = nil)
     end
   end
 
+  def self.encode(value, key = nil)
```

Comment:
@nirmaoz wdyt?

Reply:
1. how about we generalize this? something like a list of key prefixes, for which we do not encode the value.
2. is filter[... the only prefix we do not want to encode?
3. why do we not want to encode some values? are they passed already encoded?